models/nft/nft_transfers.sql

```diff
@@ -0,0 +1,124 @@
+{{ config(
+        alias ='transfers',
+        partition_by='block_date',
+        materialized='incremental',
+        file_format = 'delta',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
And here becomes multi-chains.
```suggestion
        post_hook='{{ expose_spells(\'["ethereum", "bnb", "avalanche_c", "gnosis", "optimism", "arbitrum", "polygon"]\',
```